backend/lookup/api/views.py

```diff
@@ -6,61 +6,62 @@
 from rest_framework.authentication import TokenAuthentication
 from rest_framework.permissions import IsAuthenticated
 
-from lookup.api.serializers import UserSerializer
-from lookup.models import Visitor
+from lookup.api.visitorsUtils import initializeVisitors
 
 import face_recognition
 import numpy as np
 
 class LookupViewSet(viewsets.ModelViewSet):
     """Provide CRUD + L functionality for Lookup."""
-
-    queryset = Visitor.objects.all().order_by("-created_at")
-    serializer_class = UserSerializer
     authentication_classes = [TokenAuthentication] # Authenticate access to API by issueing a token
-    permission_classes = [IsAuthenticated] # Login required
-    lookup_field = "id"
+    permission_classes = [IsAuthenticated] # Allows access only to authenticated users.
+    lookup_field = "id" # pk
+    _visitors_data = initializeVisitors() # instantiate VisitorsData class(a singleton class)
+    _visitors = _visitors_data.getVisitors() # initialize the visitors field(type: dict): {uuid: face_encodings}
+    __LAPSE = datetime.timedelta(seconds=40) # waiting time for a duplicate user
```

Comment:
lapse for the continuous visitor(so that the continuous recognition for the same visitor can be avoided)